tecken/libdockerflow.py

```diff
@@ -11,23 +12,25 @@
 from tecken.libstorage import StorageError
 
 
+logger = logging.getLogger("tecken")
```

Comment:
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1910363 to rename the loggers in Tecken.